osmomath/decimal.go

```diff
@@ -1025,15 +1025,15 @@ func MaxBigDec(d1, d2 BigDec) BigDec {
 // DecEq returns true if two given decimals are equal.
 // Intended to be used with require/assert:  require.True(t, DecEq(...))
 //
-//nolint:thelper
+//nolint:thelper, revive
```

Comment:
what does this do?

Reply:
allows 5+ return arguments. (by bringing osmoutils / osmomath into our repo, it started to get linted following our existing linting rules. Got bunch of linting problems that I needed to fix)